docs/_posts/Meryem1425/2023-05-24-sbertresolve_icd10cm_augmented_en.md

```diff
@@ -0,0 +1,172 @@
+|:--------------------------------------|:---------|:-------------|:------------------------------------------------------------------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| gestational diabetes mellitus         | PROBLEM  | O24.4        | O24.4:::O24.41:::O24.43:::Z86.32:::K86.8:::P70.2:::O24.434:::E10.9                  | gestational diabetes mellitus:::gestational diabetes mellitus:::gestational diabetes mellitus in the puerperium:::history of gestational diabetes mellitus:::secondary pancreatic diabetes mellitus:::neonatal diabetes mellitus:::gestational diabetes mellitus in the puerperium, insulin controlled:::juvenile onset diabetes mellitus                                                                                                                            |
+| subsequent type two diabetes mellitus | PROBLEM  | E11          | E11:::E11.9:::E10.9:::E10:::E13.9:::Z83.3:::L83:::E11.8:::E11.32:::E10.8:::Z86.39   | type 2 diabetes mellitus:::type ii diabetes mellitus:::type i diabetes mellitus:::type 1 diabetes mellitus:::secondary diabetes mellitus:::fh: diabetes mellitus:::type 2 diabetes mellitus with acanthosis nigricans:::complication of type ii diabetes mellitus:::secondary endocrine diabetes mellitus:::complication of type i diabetes mellitus:::history of diabetes mellitus type ii:::pregnancy and type 2 diabetes mellitus                                 |
+| HTG-induced pancreatitis              | PROBLEM  | M79.3        | M79.3:::F10.2:::K85.3:::T46.5X:::K85.20:::K85.32:::T50.90:::A39.8:::J98.5           | drug-induced panniculitis:::drug-induced chronic pancreatitis:::drug-induced acute pancreatitis:::drug-induced pericarditis:::alcohol-induced pancreatitis:::drug induced acute pancreatitis with infected necrosis:::drug-induced dermatomyositis:::rbn - retrobulbar neuritis:::drug-induced granulomatous mediastinitis                                                                                                                                           |
```

Comment:
The results should be all correct, so please change this chunk. "HTG-induced pancreatitis" can be ;

- drug-induced pancreatitis
- pancreatitis

Reply:
Done